examples/example_parameters.param

```diff
@@ -0,0 +1,106 @@
+numTemplates 0
+# List of images to skip. Enter as space separated indexes between the curly braces e.g. { 10 20 42 }, or use a list in the python wrapper.
+templateSkipNums {  }
+
+######################
+### Pre-processing ###
+######################
+# Smooth the images with a gaussian. If 0 then the images are not smoothed. Otherwise the sigma of the gaussian is determined by this parameter.
+preSmoothSigma 0
+# Map the intensity to the range 0-1 by default (0). If 1 then don't do that.
+dontNormalizeInputImages 0
+# Enhance contrast of the images mainly to cut off outliers.
+enhanceContrastSaturationPercentage 0.15
+# Minimum intensity in the image is mapped to 0
+normalizeMinToZero 1
+
+###################
+### Calculation ###
+###################
+# Rigid registration
+# ------------------
+# Pre-align images with rigid registration (translation) using cross-correlation
+useCorrelationToInitTranslation 0
+# The maximum translation in pixels of each image in this alignment. 0 = no limit on the translation
+maxCorrShift 0
+
+# Gradient descent settings
+# -------------------------
+# Maximum number of gradient descent steps
+maxGDIterations 500
+# Desired "precision" to be achieved during gradient descent
+stopEpsilon 1e-6
+# Regularization factor in the first stage, highest level
+lambda 200
+# Regularization factor update per level. For example, if you have images of level 8, then lambda will be applied in 8, lambda * lambdaFactor in 7, lambda * lambdaFactor^2 in 6...
+lambdaFactor 1
+
+# Levels
+# ------
+# Levels specify the resolution of the input images and the parameters of the multilevel minimization. The multilevel algorithm expects the input images to be quadratic and to have a dyadic resolution, i.e. the number of pixels in one coordinate direction is a power of two. The exponent is called level. For instance, level 8 means that the input images have $2^8=256$ pixels in each direction. Thus, 9 and 10 encode 512 and 1024 pixels respectively.
+# Most of these should be filled out by default if you use the python wrapper.
+# The precision of the input images = log_2(image sidelength) and the size of the output
+precisionLevel 8
+# The stopping level of the grid. i.e. deformations are not calculated beyond this resolution even if images have more pixels. If you want the resolution to equal the resolution of the images, stoplevel = precisionLevel. Higher levels take longer to compute.
+stopLevel 8
+# the starting level of the grid, i.e. the optimization will first be done on a grid of size 2^startLevel x 2^startlevel. startLevel should be small enough so that you still see the individual atoms if the input images are downsampled to this size. Usually startLevel is one or two smaller than stopLevel.
+startLevel 6
+# In the refinement step start at this level, usually >startLevel and <=stopLevel.
+refineStartLevel 7
+# In the refinement step stop at this level
+refineStopLevel 8
+# ?
+checkboxWidth 8
```

Comment:
This controls the size of the stripes in the fine_checkview_**.png images, which are meant to visualize the registration accuracy.